packages/concerto-core/lib/introspect/metamodel.js

```diff
@@ -367,11 +368,15 @@ class MetaModel {
      * Resolve the namespace for names in the metamodel
      * @param {object} modelManager - the ModelManager
      * @param {object} metaModel - the MetaModel
+     * @param {boolean} [validate] - whether to perform validation
```

Comment:
This is just syntactic/structural validation, right?

Reply:
Well it's Concerto validation of the JSON as an instance of the metamodel. We do use "validation" in possibly different ways throughout the code. There are things that validating the model requires that isn't checked here though e.g., no cycles in the type hierarchy.